src/components/SwipeButton/index.js

```diff
@@ -217,6 +220,7 @@ SwipeButton.propTypes = {
   ]),
   thumbIconStyles: PropTypes.object,
   title: PropTypes.string,
+  titleMaxFontScale: PropTypes.number,
```

Comment:
Can you please add this newly introduced prop in documentation i.e `README.md` & `docs/index.html`

Reply:
will do